src/resource/page/ResourcePage.tsx

```diff
@@ -152,9 +135,11 @@ export default function ResourcePage({ id }: { id: string }): JSX.Element {
         </p>
       </ResourceDetailsSection>
       <ResourceSourceLink id="resource-source-link" resource={resource} />
-      <ResourceSection id="resource-opening-hours">
-        <ResourceOpeningHours id={id} datePeriods={datePeriods} />
-      </ResourceSection>
+      {resource && (
+        <ResourceSection id="resource-opening-hours">
```

Comment:
Not related to this PR, but I think the ratio of boilerplate/beef in this `ResourceSection` component is quite large. It basically returns html tag section with id and classname. Though it is applied in 2 place so I guess having it as a separate component is justified 🤔